src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+:root {
+  --font-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__link {
+  color: black;
+  text-decoration: none;
+  display: flex;
+  padding: 23px 0;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.nav__link:hover {
+  color: var(--font-color);
+}
+
+.nav__item {
+  margin-left: 20px;
```

Comment:
Pay attention to first child. It shouldn't have a margin. Use pseudo-class `:not(:first-child)`